### PR TITLE
Fix/collate conflict bug

### PR DESCRIPTION
**Pull request by carissadean**
Sorts revisions by timestamp, with the most recent revision being first.

More info: https://snyksec.atlassian.net/browse/ELK-709

### CODE REVIEW

File: vervet-underground/internal/storage/revision.go [hunk 1]
```diff
@@ -75,3 +75,16 @@ func (s ServiceRevisions) ResolveLatestRevision(version vervet.Version) (Content
 	}
 	return revision, nil
 }
+
+type Revisions []ContentRevision
```
Comment (cmars) [inline vervet-underground/internal/storage/revision.go:79]: Suggest naming this `ContentRevisions` for a clearer association with the slice element type.

File: vervet-underground/internal/storage/revision.go [hunk 2]
```diff
@@ -75,3 +75,16 @@ func (s ServiceRevisions) ResolveLatestRevision(version vervet.Version) (Content
...
+
+// Less implements sort.Interface.
+func (r Revisions) Less(i, j int) bool {
+	return r[i].Timestamp.After(r[j].Timestamp)
```
Comment (cmars) [inline vervet-underground/internal/storage/revision.go:83]: Suggest making the ordering deterministic even when timestamps aren't unique. The `Digest` would be a good tie-breaker if the timestamps are equal.

Seems like a remote edge case, but time sometimes runs strangely within VMs and containers.

**Review by cmars — APPROVED** [review]
LGTM with a couple of suggestions. I may pick these up this afternoon & land if you're OK with it.

**Comment by carissadean** [comment]
> LGTM with a couple of suggestions. I may pick these up this afternoon & land if you're OK with it.

Yes feel free! @cmars

**Pull request merged**